test/util/aleTests/ALEyvyCRVHelperForkTest.t.sol

```diff
@@ -267,20 +218,20 @@ contract ALEHelperForkTest is Test {
         assertEq(DOLA.balanceOf(userPk), 0);
     }
 
-    function test_depositAndLeveragePosition() public {
-        // vm.assume(styCRVAmount < 7900 ether);
-        // vm.assume(styCRVAmount > 0.00000001 ether);
+    function test_depositAndLeveragePosition(uint256 styCRVAmount) public {
+        vm.assume(styCRVAmount < 7900 ether);
```

Comment:
Why is styCRVAmount so low in this fuzz test?

Reply:
Is the best approach to use the balance of an EOA as the fuzz limit? The fuzz test should not be limited by what some arbitrary holder has, but rather by constraints that make sense in terms of the contract code. Is there a reason for not using something like `vm.deal(styCrv, testUser, fuzz_amount )`?